client/src/components/Cards/__tests__/__snapshots__/index.spec.js.snap

```diff
@@ -11,27 +11,9 @@ exports[`components/<Cards /> should render component with data 1`] = `
       }
     }
   >
-    <CardItem
-      data={
-        Object {
-          "id": "id-1",
-        }
-      }
-    />
-    <CardItem
-      data={
-        Object {
-          "id": "id-2",
-        }
-      }
-    />
-    <CardItem
-      data={
-        Object {
-          "id": "id-3c",
-        }
-      }
-    />
+    <CardItem />
```

Comment:
Do not really think it is correct to have such snapshots. I think is better to mock some stupid data and save a snapshot with some non-empty props...